test/unittest/CMakeLists.txt

```diff
@@ -142,8 +135,7 @@ foreach(blas_test ${SYCL_UNITTEST_SRCS})
   target_link_libraries(${test_exec} PRIVATE gtest_main Clara::Clara blas::blas portblas)
   target_include_directories(${test_exec} PRIVATE ${CBLAS_INCLUDE} ${PORTBLAS_COMMON_INCLUDE_DIR})
 
-  list (FIND SYCL_UNITTEST_NOFASTMATH ${blas_test} _index)
-  if (${_index} GREATER -1)
+  if (${CMAKE_CXX_COMPILER_ID} STREQUAL "IntelLLVM")
```

Comment:
Thanks for your PR, @s-Nick.

Please, could you add a comment stating that this condition is true just in case of the `icpx` release and not for `intel/llvm`?

Reply:
Thank you for your review @pgorlani 
I addressed your comments in this commit 7b2c73a2875ac96e0d649230fd0b68b8f266992e